lib/diagnostic.rb

```diff
@@ -18,7 +18,8 @@ def example
 # In a Ruby comment, explain Behavior Driven Development, how it is meant to be
 # used, and how it differs from Test Driven Development.
 
-# your answer here
+BDD is a development technique that encourages collaboration, BDD has evolved
+from TDD and is geting everyone on the develpment team involed and paticipating.
 
```

Comment:
Yes, but it's mostly because we're testing behavior and not just running arbitrary tests, this makes BDD read more easily and feel more impactful.